src/main/java/com/epam/cdp/m2/hw2/aggregator/fjp/SumListRecursiveTask.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.cdp.m2.hw2.aggregator.fjp;
+
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.collections4.ListUtils;
+
+import java.util.List;
+import java.util.concurrent.RecursiveTask;
+
+@RequiredArgsConstructor
+public class SumListRecursiveTask extends RecursiveTask<Integer> {
```

Comment:
Just question. We have also `Spliterator` interface. In which way it could be helpful in this context. `Spliterator` vs `RecursiveTask`?